### PR TITLE
#26, #87, #187: AtlasCheckout initialization using Info.plist:

### PR DESCRIPTION
**Pull request by artdaw**
- ATLASSDK_INTERFACE_LANGUAGE key is added to be used in Info.plist.
- AtlasCheckout initialization using Info.plist added.
- All necessary information about AtlasCheckout initialization using Info.plist added to the README. Some links to images fixed.
- Unnecessary Documentation generated by Jazzy removed.

### CODE REVIEW

**Comment by hani-ibrahim** [comment]
👍

File: README.md [hunk 1]
```diff
@@ -3,8 +3,6 @@
 [![CocoaPods](https://img.shields.io/cocoapods/v/AtlasSDK.svg?maxAge=3600)](http://cocoadocs.org/docsets/AtlasSDK)
 [![CocoaPods](https://img.shields.io/cocoapods/p/AtlasSDK.svg?maxAge=3600)](http://cocoadocs.org/docsets/AtlasSDK)
 [![CocoaPods](https://img.shields.io/cocoapods/at/AtlasSDK.svg?maxAge=3600)](http://cocoadocs.org/docsets/AtlasSDK)
-
-[![Build Status](https://travis-ci.org/zalando-incubator/atlas-ios.svg?branch=master)](https://travis-ci.org/zalando-incubator/atlas-ios)
```
Comment (bonkey) [inline README.md]: Pity, but that's the way go... for now. I really liked Mr. travis.

File: README.md [hunk 2]
```diff
@@ -76,9 +78,36 @@ override func viewDidLoad() {
 }
 ```
 
+### Configuration using Info.plist
+
+You can provide all configuration options application wide using `Info.plist` with the following keys:
+- ATLASSDK_CLIENT_ID: String - Client Id (required)
+- ATLASSDK_SALES_CHANNEL: String - Sales Channel (required)
+- ATLASSDK_USE_SANDBOX: Bool - Indicates whether sandbox environment should be used
+- ATLASSDK_INTERFACE_LANGUAGE: String - Checkout interface language
```
Comment (bonkey) [inline README.md]: Maybe some formatting here? Like *ATLASSDK_INTERFACE_LANGUAGE* or `ATLASSDK_INTERFACE_LANGUAGE`

File: README.md [hunk 3]
```diff
@@ -76,9 +78,36 @@ override func viewDidLoad() {
...
+
+var atlasCheckout: AtlasCheckout?
+
+override func viewDidLoad() {
+	super.viewDidLoad()
+
+	AtlasCheckout.configure { result in
+		if case let .success(checkout) = result {
+    	self.atlasCheckout = checkout
+    }
```
Comment (bonkey) [inline README.md]: Some indenting.

File: README.md [hunk 4]
```diff
@@ -132,6 +161,9 @@ AtlasSDK framework is the core framework (think about Cocoa Touch Foundation) th
 AtlasUI uses AtlasSDK as the foundation and provides public checkout functionality to customers.
 AtlasUI also contains all the generic UI methods used internally within Atlas SDK. Think about it as Cocoa Touch UIKit.
 
+## AtlasSDK Documentation
+
+[AtlasSDK Reference Documentation](http://cocoadocs.org/docsets/AtlasSDK) is generated automatically during the Pod deployment.
```
Comment (bonkey) [inline README.md]: 💯

**Comment by bonkey** [comment]
👍

File: README.md (hunk 2 above)
Comment (artdaw) [inline README.md]: Agree I like second variant

File: README.md (hunk 3 above)
Comment (artdaw) [inline README.md]: Fixed

**Comment by ahmed-shehata** [comment]
👍

**Comment by artdaw** [comment]
👍

**Pull request merged**